xml/Microsoft.Win32/FileDialog.xml

```diff
@@ -324,9 +324,10 @@
       <MemberSignature Language="C#" Value="public event System.ComponentModel.CancelEventHandler FileOk;" />
       <MemberSignature Language="ILAsm" Value=".event class System.ComponentModel.CancelEventHandler FileOk" />
       <MemberSignature Language="DocId" Value="E:Microsoft.Win32.FileDialog.FileOk" />
-      <MemberSignature Language="VB.NET" Value="Public Event FileOk As CancelEventHandler " />
+      <MemberSignature Language="VB.NET" Value="Public Event FileOk As CancelEventHandler " FrameworkAlternate="netcore-3.0;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
```

Comment:
similar question here but for VB. Only VB has a different signature for different versions

Reply:
This version changes the way that signatures are added to the type and member nodes ... before, it would only break out into different a `FrameworkAlternate` when one of the core attributes changes (param name, base type, etc). However now, it will break out the signature based on **any difference**, regardless of whether it causes `FrameworkAlternate` to be added to a core attribute.

So what this means is that before, some differences were being lost ... now, the individual signature formatters could find small differences between versions. It could also highlight issues with *other* formatters that might not support some given difference (for example in this case, I'm not that familiar with VB.NET to know what the `Custom` keyword does, and whether the other signatures also need to have a different representation).

Going forward, we can treat these as individual bug reports or feature requests for formatters ... so for example we might say, "VB formatter should not add `Custom` keyword", **or** "C# formatter should add `custom` keyword" (again, that's just an example).